core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java

```diff
@@ -407,6 +410,17 @@ private static boolean hasFakeSequence(PoolableElementView view) {
         .anyMatch(f -> f.hasFakeSequence());
   }
 
+  @Override
+  public boolean hasViewWithoutIndex() {
```

Comment:
Even though these are actually on the views, they're really about the libraries, so I think this should be `hasLibraryWithoutIndex`. So, if there's only one, it's fine if it has no index. So, add a guard of `if (view.size() < 2) return false;`.